apps/src/applab/ImportScreensDialog.jsx

```diff
@@ -193,26 +197,34 @@ var ImportScreensDialog = React.createClass({
   },
 
   render() {
+    if (!this.props.project) {
+      return null;
+    }
     const nonImportableScreens = this.props.project.screens.filter(s => !s.canBeImported);
+    const importableScreens = this.props.project.screens.filter(s => s.canBeImported);
+    const canImport = importableScreens.length > 0 || this.props.project.otherAssets.length > 0;
     return (
       <Dialog
         title={`Import from Project: ${this.props.project.name}`}
-        confirmText="Import"
-        onConfirm={() => this.props.onImport(this.state.selectedScreens,
-                                               this.state.selectedAssets)}
+        confirmText={canImport ? "Import" : null}
+        onConfirm={canImport ? () => this.props.onImport(this.state.selectedScreens,
+                                                         this.state.selectedAssets)
+                                 : null}
+        onCancel={canImport ? null : this.props.handleClose}
```

Comment:
A lot of stuff is `null` if `canImport` is false.  What is the behavior of this dialog if there are no importable screens or assets?  Should it just be rendering a whole different Dialog component in that case?